js/csrfprotector.js

```diff
@@ -19,7 +19,10 @@ var CSRFP = {
 	 *
 	 * @var string array
 	 */
-	checkForUrls: [],
+	checkForUrls: Array.prototype.slice.call(document.getElementsByName("checkForUrls"))
```

Comment:
**NOTE**: See below comment first

Write a method in init method to check if ELEMENT [NAME = 'checkForUrls'] exists and load only then.
Also using `checkForUrls` can be risky it could interfere with original code of the user, use something line `CSRFP_checkForUrls`